src/DCRoom.php

```diff
@@ -158,6 +158,33 @@ public function showForm($ID, array $options = [])
         echo "</td>";
         echo "</tr>";
 
+        echo "<tr class='tab_bg_1'>";
+        echo "<td><label for='dropdown_vis_cell_width$rand'>" . __('Cell width') . "</label></td><td>";
+        Dropdown::showNumber(
+            "vis_cell_width",
+            [
+                'value'  => $this->fields["vis_cell_width"],
```

Comment:
Default value should be proposed on creation.

```suggestion
                'value'  => $this->fields["vis_cell_width"] ?: 40,
```